src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+}
+
+body {
+  margin: 0 8px;
+}
+
+.search-form__lable {
+  display: block;
+  position: relative;
+}
+
+.search-form__lable::before {
+  content: "";
+  position: absolute;
+
+  background-image: url("./images/Search.svg");
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.search-form__field {
+  display: block;
+  font-family: inherit;
+  width: 100%;
+  outline: none;
+  border-radius: 4px;
+  color: inherit;
+  font-size: inherit;
+  font-weight: inherit;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  transition: box-shadow 0.3s ease 0s;
+  box-sizing: border-box;
+}
+
+.search-form__field:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-form__field::placeholder:focus,
```

Comment:
The `search-form__field::placeholder:focus` selector is invalid. CSS doesn't provide any way to style the placeholder on focus.